lib/dream_html.ml

```diff
@@ -865,3 +864,101 @@ module Hx = struct
   let ws_connect fmt = string_attr "data-ws-connect" fmt
   let ws_send = attr "data-ws-send"
 end
+
+module MathML = struct
+  (* Attributes *)
+  let accent = bool_attr "accent"
+  let accentunder = bool_attr "accentunder"
+  let columnspan = int_attr "columnspan"
+  let depth fmt = string_attr "depth" fmt
+
+  let display value =
+    ( "display",
+      match value with
+      | `block -> "block"
+      | `inline -> "inline" )
+
+  let displaystyle = bool_attr "displaystyle"
+  let fence = bool_attr "fence"
+  let framespacing fmt = string_attr "framespacing" fmt
```

Comment:
Looks like this is non-standard and not supported by most browsers: https://developer.mozilla.org/en-US/docs/Web/MathML/Element/mtable#browser_compatibility

Can you review all the attrs one more time to check that only the standard ones are included here? Thx.